src/lib/AlphaBanner.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  export let contacts: [string, string][] = [];
+</script>
+
+<div class="govuk-phase-banner">
+  <p class="govuk-phase-banner__content">
+    <strong class="govuk-tag govuk-phase-banner__content__tag">Alpha</strong>
+    <span class="govuk-phase-banner__text">
+      This is a new service. 
+      {#if contacts.length > 0} 
```

Comment:
This is missing commas for the demo with 3 people. Instead of making it general right now, since we are the only users of this library, how about just hardcoding our names/emails like we have in existing tools?